src/adapters/postgres/fields-adapter.ts

```
@@ -2201,6 +2224,13 @@ export class PostgresFieldsService implements IServicelocatorfields {
       );
     }
 
+    // Fetch the canonical country list once (not per row), only when this
+    // user actually has a country-type field to parse.
+    const hasCountryField = result.some((data) =>
+      data?.label?.toLowerCase().includes('country')
```

ESLint

[error] Replace `'country'` with `"country"`

(prettier/prettier)